packages/dapp/src/i18n/en/translation.json

```diff
@@ -32,7 +32,8 @@
       "switchChain": "Switch Chain",
       "copy": "Copy",
       "explore": "Explore",
-      "copiedMsg": "Wallet address copied"
+      "copiedMsg": "Wallet address copied",
```

Comment:
Please don't shorten keys -> copiedMessage 🙂

Reply:
I will keep that in mind from now on when creating new translation-keys. Also here, it had been like this ever since I created that task of "Copying a wallet address". So, we should probably stick with it now to avoid crowdin conflicts, right?